backend/src/main/java/com/google/capstone/UploadInformation.java

```diff
@@ -0,0 +1,92 @@
+package com.google.capstone;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import com.google.capstone.Information;
+
+import java.io.IOException;
+import java.lang.String;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// Servlet that stores art information in a datastore and retrieves that data to conver to json
+@WebServlet("/api/v1/uploadInfo")
+public class UploadInformation extends HttpServlet {
+  
+  /** 
+   * doGet funtion that gets the information from the data store and 
+   * then converts the data to json format. Currently there is no fetch function that uses this.
+   */
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("ImageInformation").addSort("timestamp", SortDirection.DESCENDING);
+   
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<Information> infos = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
```

Comment:
Optional: Could abstract this to a converter helper method (Entity to Information) and use Java Streams. Did something similar in portfolio based on our host's advice: [here](https://github.com/indhu-gunda/my-portfolio/blob/master/portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java#L58)